projects/otelarrow.md

```diff
@@ -74,7 +74,11 @@ Project approvers
 
 ## Meeting times
 
-Every other Thursday at 8AM PT.
+Bi-weekly Thursday at 8AM PT.
+
+Bi-weekly Tuesday at 4PM PT.
+
+See the [Calendar](../README.md#calendar) to find upcoming meetings.
 
```

Comment:
https://github.com/open-telemetry/community?tab=readme-ov-file#implementation-sigs Can you update the table here too?